catalog.html

```diff
@@ -1,118 +1,112 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
 
 <head>
-  <meta charset="utf-8">
-  <title>HTML Academy: Седона</title>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&display=swap" rel="stylesheet">
+    <link href="normalize.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
 </head>
 
-<body>
+<body class="page-body">
 
 </body>
-<header>
-
-  <nav>
-    <a href="">
-      <img src="https://via.placeholder.com/150x100" alt="">
-    </a>
-    <ul>
-      <li><a href="">информация</a></li>
-      <li><a href="">фото и видео</a></li>
-      <li><a href="">карта штата</a></li>
-      <li><a href="">гостиницы</a></li>
-    </ul>
-  </nav>
+<header class="container">
+    <nav class="main-nav">
+        <a class="main-nav-logo" href="">
+            <img src="https://via.placeholder.com/150x100" alt="">
+        </a>
+        <ul class="main-nav-list list-reset">
+            <li class="main-nav-link"><a href="">информация</a></li>
+            <li class="main-nav-link"><a href="">фото и видео</a></li>
```

Comment:
li ccылкой не является
```suggestion
            <li class="main-nav-item"><a class="main-nav-link" href="">фото и видео</a></li>
```